traffic_control/locale/fi/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,331 @@
+msgid "Average"
+msgstr "Tyydyttävä"
+
+#: traffic_control/models.py:61
+msgid "Good"
+msgstr "Hyvä"
+
+#: traffic_control/models.py:62
+msgid "Very good"
+msgstr "Erittäin hyvä"
+
+#: traffic_control/models.py:69
+msgid "Code"
+msgstr "Merkkikoodi"
+
+#: traffic_control/models.py:71 traffic_control/models.py:89
+msgid "Description"
+msgstr "Kuvaus"
+
+#: traffic_control/models.py:76
+msgid "Traffic Sign Code"
+msgstr "Merkkikoodi"
+
+#: traffic_control/models.py:77
+msgid "Traffic Sign Codes"
+msgstr "Merkkikoodit"
+
+#: traffic_control/models.py:87
+msgid "Status"
+msgstr "Tila"
+
+#: traffic_control/models.py:94
+msgid "Lifecycle"
+msgstr "Elinkaari"
+
+#: traffic_control/models.py:95
+msgid "Lifecycles"
+msgstr "Elinkaaret"
+
+#: traffic_control/models.py:105 traffic_control/models.py:200
+msgid "Location (2D)"
+msgstr "Sijainti (2D)"
+
+#: traffic_control/models.py:107 traffic_control/models.py:202
+msgid "Height"
+msgstr "Korkeus"
+
+#: traffic_control/models.py:109 traffic_control/models.py:204
+msgid "Direction"
+msgstr "Suunta"
+
+#: traffic_control/models.py:112 traffic_control/models.py:207
+msgid "Traffic sign code value"
+msgstr "Liikennemerkin arvo"
+
+#: traffic_control/models.py:113 traffic_control/models.py:208
+msgid "Structure id"
+msgstr "Kiinnityskohteen ID"
+
+#: traffic_control/models.py:115 traffic_control/models.py:210
+msgid "Structure"
+msgstr "Kiinnityskohteen tyyppi"
+
+#: traffic_control/models.py:121 traffic_control/models.py:216
+msgid "Created at"
+msgstr "Luotu-aikaleima"
+
+#: traffic_control/models.py:122 traffic_control/models.py:217
+msgid "Updated at"
+msgstr "Muokattu-aikaleima"
+
+#: traffic_control/models.py:123 traffic_control/models.py:218
+msgid "Deleted at"
+msgstr "Poistettu-aikaleima"
+
+#: traffic_control/models.py:142 traffic_control/models.py:237
+msgid "Validity period start"
+msgstr "Voimassaoloaika alkaa"
+
+#: traffic_control/models.py:145 traffic_control/models.py:240
+msgid "Validity period end"
+msgstr "Voimassaoloaika loppuu"
+
+#: traffic_control/models.py:148 traffic_control/models.py:243
+msgid "Seasonal validity period start"
+msgstr "Kausiluontoinen voimassaoloaika alkaa"
+
+#: traffic_control/models.py:151 traffic_control/models.py:246
+msgid "Seasonal validity period end"
+msgstr "Kausiluontoinen voimassaoloaika loppuu"
+
+#: traffic_control/models.py:153 traffic_control/models.py:248
+msgid "Owner"
+msgstr "Omistaja"
+
+#: traffic_control/models.py:154 traffic_control/models.py:253
+msgid "Txt"
+msgstr "Lisätieto"
+
+#: traffic_control/models.py:155
+msgid "Decision date"
+msgstr "Päätöspvm"
+
+#: traffic_control/models.py:157 traffic_control/models.py:268
+msgid "Decision id"
+msgstr "Päätöksen tunnus"
+
+#: traffic_control/models.py:159
+msgid "Plan link"
+msgstr "Suunnitelmalinkki"
+
+#: traffic_control/models.py:161 traffic_control/models.py:270
+msgid "Size"
+msgstr "Koko"
+
+#: traffic_control/models.py:164 traffic_control/models.py:273
+msgid "Reflection"
+msgstr "Kalvo"
```

Comment:
This seems wrong. Not sure if there's some industry terminology I'm not aware of but pointing this out just in case.

Reply:
It's industry term